cmake/FindRust.cmake

```diff
@@ -190,6 +194,7 @@ function(cargo_vendor)
     # for online builds when we run `cpack --config CPackSourceConfig.cmake`
     message(STATUS "Running `cargo vendor` to collect dependencies for ${ARGS_TARGET}. This may take a while if the local crates.io index needs to be updated ...")
     make_directory(${CMAKE_SOURCE_DIR}/.cargo)
+    GetCargoLocalDirEnv(_cargo_local_dir_env)
```

Comment:
Two things:
1. It would be nice to add some new lines between these 4 function calls.  
2. Do we need to call `GetCargoExtraEnv(_cargo_extra_env)` as well?  Probably not, but I thought I should ask.

Reply:
1. Done.
2. Tricky.  `execute_process()` doesn't support the `COMMAND_EXPAND_LISTS` directive, which is necessary to make the environment supplementation work properly. You also didn't reference the other variable here (the include dir), so I just left this one as-is.